### PR TITLE
Governance spec update

### PR DESCRIPTION
**Pull request by grarco**
- Fixes typos
- Updates vote storage key

### CODE REVIEW

File: src/architecture/governance/signaling.md [hunk 1]
```diff
@@ -137,27 +138,31 @@ struct OnChainVote {
...
 ```
 
 The storage key will only be created if the transaction is signed either by a validator or a delagator. 
-Validators will be able to vote only for 2/3 of the total voting period, meanwhile delegators can vote until the end of the voting period.
-If a delegator votes opposite to its validator this will *overri*de the corresponding vote of this validator (e.g. if a delegator has a voting power of 200 and votes opposite to the delegator holding these tokens, than 200 will be subtracted from the votig power of the involved validator).
+Validators will be able to vote only for 2/3 of the total voting period, while delegators can vote until the end of the voting period.
+
+If a delegator votes opposite to its validator, this will *override* the corresponding vote of this validator (e.g. if a delegator has a voting power of 200 and votes opposite to the delegator holding these tokens, than 200 will be subtracted from the votig power of the involved validator).
+
+If a delegator votes accordingly to its validator, unless the transaction is being forced, the vote will not actually be submitted to the chain, as a small form of space optimization.
```
Comment (cwgoes) [inline src/architecture/governance/signaling.md:149]: but what if the delegator votes prior to the 2/3 mark, where the validator could still change their vote? 

(or is this rendered impossible somehow?)

**Review by cwgoes — CHANGES_REQUESTED** [review]
Thanks, just one clarifying question

File: src/architecture/governance/signaling.md (hunk 1 above)
Comment (Fraccaman) [reply src/architecture/governance/signaling.md:149]: yes, you are right. Without using the `--force` option the result depends if the validator voted or not. Still, delegators have the last 1/3 of the voting period to re-submit their vote. Do you think we should always send the votes?

Comment (Fraccaman) [reply src/architecture/governance/signaling.md:149]: @grarco is suggesting that we could use the *optimized behaviour* only if delegators are submit a vote transaction during the ending 1/3 voting period

Comment (cwgoes) [reply src/architecture/governance/signaling.md:149]: I agree with that suggestion. Default to safe :)

Comment (grarco) [reply src/architecture/governance/signaling.md]: Specs updated to clarify this scenario

**Comment by Fraccaman** [comment]
LGTM!

**Review by Fraccaman — APPROVED** [review]

**Review by cwgoes — APPROVED** [review]

**Comment by cwgoes** [comment]
Thank you!

**Pull request merged**